.ebextensions/cwl-webrequest-metrics.config

```diff
@@ -0,0 +1,193 @@
+###############################################################################
+## Option Settings
+##    Namespace: "aws:elasticbeanstalk:application:environment"
+##    OptionName: WebRequestCWLogGroup
+##       Default:   <EnvironmentName>-webrequests
+##    Description: This is the name of the cloudwatch log group for web requests (access log)
+##
+## To get an SNS alert, add a subscription to the Elastic Beanstalk Notification
+##   topic.  (e.g. set your Notificiation Endpoint to your email address)
+##
+## Metrics
+##  Namespace:  ElasticBeanstalk/<EnvironmentName>
+##  Metrics:    CWLHttp4xx  CWLHttp5xx
+##
+## Cloudwatch Alarms
+##   CWLHttp5xxCount   - this alarm fires if the number of calls returning
+##                        5xx response codes > 10
+##   CWLHttp4xxPercent - this alarm fires if the percentage of calls returning
+##                        4xx response codes > 10%
+##
+###############################################################################
+
+# Nginx access log pattern:
+# $remote_addr - $remote_user [$time_local] "$request" $status $body_bytes_sent "$http_referer" "$http_user_agent" "$http_x_forwarded_for"
+Mappings:
+  CWLogs:
+    WebRequestLogGroup:
+      LogFile: "/var/log/nginx/access.log"
+      TimestampFormat: "%d/%b/%Y:%H:%M:%S %z"
+    FilterPatterns:
+      Http4xxMetricFilter: "[..., status=4*, size, referer, agent, xforward]"
+      HttpNon4xxMetricFilter: "[..., status!=4*, size, referer, agent, xforward]"
+      Http5xxMetricFilter: "[..., status=5*, size, referer, agent, xforward]"
+      HttpNon5xxMetricFilter: "[..., status!=5*, size, referer, agent, xforward]"
+    RailsProductionLogGroup:
+      LogFile: "/var/log/eb-docker/containers/eb-current-app/production.log"
+      TimestampFormat: "%d/%b/%Y:%H:%M:%S %z"
+
+Outputs:
+  WebRequestCWLogGroup:
+    Description: "The name of the Cloudwatch Logs Log Group created for this environments web server access logs. You can specify this by setting the value for the environment variable: WebRequestCWLogGroup. Please note: if you update this value, then you will need to go and clear out the old cloudwatch logs group and delete it through Cloudwatch Logs."
+    Value: { "Ref" : "AWSEBCloudWatchLogs8832c8d3f1a54c238a40e36f31ef55a0WebRequestLogGroup" }
+  RailsProductionCWLogGroup:
+    Description: "Log group for the production.log file containing Rails log stream"
+    Value: { "Ref" : "AWSEBCloudWatchLogs8832c8d3f1a54c238a40e36f31ef55a0RailsProductionLogGroup" }
```

Comment:
I'm confused by this prefix, but it says below that this must be the prefix. Beats me, really. The docs are bad :/ Will we need to set up this resource differently for staging vs. production vs. testing?